packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -144,26 +144,6 @@ contract Deploy is Deployer {
         return keccak256(bytes(Config.implSalt()));
     }
 
-    /// @notice Returns the proxy addresses. If a proxy is not found, it will have address(0).
-    function _proxies() internal view returns (Types.ContractSet memory proxies_) {
```

Comment:
This function requires that a proxy exists, which does not work with the approach of deploying the L2OO after the call to `deployOpChain`. It did not give us much additional safety, since the output is usually passed to `ChainAssertions` which ensures the contracts exist by making calls to them.